### ISSUE TITLE
[Bug]: Bottle name with spaces is incorrect in generated desktop entry file

### ISSUE THREAD

**Issue by mphojele** [bottlesdevs/Bottles#1890]
### Describe the bug

Games for which i generated a desktop entry file will not launch if they are in a bottle that has a name with a space e.g. "Plain Cola".

### To Reproduce

1. Create a bottle with a name with a space in the middle like, "Plain Cola" and add a game/program to it
2. Make Bottles generate a desktop entry for the program in the bottle
3. Try to launch the program and program will not launch

### Package

Flatpak from Flathub

### Distribution

Fedora 36

### Troubleshooting Logs

```shell
Version: 2022.7.28-brescia-2
Display:
    X.org: true
    X.org (port): :99.0
    Wayland: true
Graphics:
    vendors:
        amd:
            vendor: amd
            envs:
                DRI_PRIME: '1'
            icd: /usr/lib/x86_64-linux-gnu/GL/vulkan/icd.d/radeon_icd.x86_64.json:/usr/lib/i386-linux-gnu/GL/vulkan/icd.d/radeon_icd.i686.json
    prime:
        integrated: null
        discrete: null
Kernel:
    Type: Linux
    Version: 5.18.13-200.fc36.x86_64
Distro:
    Name: GNOME
    Version: '"42 (Flatpak runtime)"'
Disk:
    Total: 5183696896
    Free: 5183549440
RAM:
    MemTotal: 9.7GiB
    MemAvailable: 4.9GiB
Bottles_envs: null
```

### Display Protocol

Wayland

### Display Backend used by Bottles

Wayland

**Comment by mphojele** [comment]
Possible code bug https://github.com/bottlesdevs/Bottles/blob/6d4b4d537b0636c9a2f291253c3d2a636e1c3719/src/backend/utils/manager.py#L245

**Comment by mirkobrombin** [comment]
> Possible code bug
> 
> https://github.com/bottlesdevs/Bottles/blob/6d4b4d537b0636c9a2f291253c3d2a636e1c3719/src/backend/utils/manager.py#L245
> 
> We probably want `config.get("Name")` for the option `-b`

I confirm this
https://github.com/bottlesdevs/Bottles/blob/3ef4024fa2213abba405485aa5ac07fcff4c63b1/src/backend/managers/manager.py#L792

**Comment by ak-42** [comment]
@mirkobrombin I am still getting this bug in `2022.8.14-brescia`. 

I tried to test the fix for #1855, and it did not work. The 'Add to Steam' action generated a Steam entry with the following command: `bottles:run/Ubisoft-Connect/'Trackmania'`, which failed to launch. 
But when I applied the workaround provided in the OP, editing the command like so: `bottles:run/'Ubisoft Connect'/'Trackmania'` the integration works now, and the game launches.

**Comment by mirkobrombin** [comment]
> @mirkobrombin I am still getting this bug in `2022.8.14-brescia`.
> 
> I tried to test the fix for #1855, and it did not work. The 'Add to Steam' action generated a Steam entry with the following command: `bottles:run/Ubisoft-Connect/'Trackmania'`, which failed to launch. But when I applied the workaround provided in the OP, editing the command like so: `bottles:run/'Ubisoft Connect'/'Trackmania'` the integration works now, and the game launches.

This is a different bug which hit the Steam manager.
